packages/faceted-search/src/components/Badges/BadgeMenu/BadgeMenu.component.js

```diff
@@ -31,10 +38,37 @@ export const BadgeMenu = ({
 	displayType,
 	filterBarPlaceholder,
 	t,
+	callbacks,
 	...rest
 }) => {
+	const [options, setOptions] = useState(values);
 	const currentOperators = useMemo(() => operators, [operators]);
 	const currentOperator = operator || (currentOperators && currentOperators[0]);
+	const [isLoading, setIsLoading] = useState(true);
```

Comment:
maybe better to set isLoading false by default? then no need for this line: https://github.com/Talend/ui/pull/5022/files#diff-9cd7307a01bfb12fcee969fa3d4f76874a44bf7dc27b1c37d95e3157942ee263R51

Reply:
fixed